docs-en/chapter_preface/summary.md

```diff
@@ -1,8 +1,7 @@
 # Summary
 
-- The main audience of this book is beginners in algorithm. If you already have some basic knowledge, this book can help you systematically review your algorithm knowledge, and the source code in this book can also be used as a "brushing tool library".
+- The main audience of this book is beginners in algorithm. If you already have some basic knowledge, this book can help you systematically review your algorithm knowledge, and the source code in this book can also be used as a "Coding Toolkit".
 - The book consists of three main sections, Complexity Analysis, Data Structures, and Algorithms, covering most of the topics in the field.
-- For newcomers to algorithms, it is crucial to read an introductory book in the beginning stages to take the road less traveled.
-- Animations and graphs within the book are usually used to introduce key points and difficult knowledge. These should be given more attention when reading the book.
```

Comment:
“to take the road less traveled”意味着选择不寻常或不常走的道路，这与原文的意思有所不同。原文强调的是通过阅读入门书籍可以避免常见错误，而不是走一条不常走的路。

"To take the road less traveled" means to take an unusual or less traveled road, which is different from the original meaning. The original article emphasizes that common mistakes can be avoided by reading introductory books, rather than taking a road less traveled.

Reply:
https://en.wikipedia.org/wiki/Wikipedia:Taking_the_road_less_traveled